contracts/Archetype.sol

```diff
@@ -194,14 +198,16 @@ contract Archetype is
       signature
     );
     _mint(to, quantity);
+    
+    uint256 inviteListSupply = _listSupply[auth.key];
 
     if (i.limit < i.maxSupply) {
       _minted[_msgSender()][auth.key] += quantity;
     }
     if (i.maxSupply < config.maxSupply) {
       _listSupply[auth.key] += quantity;
     }
```

Comment:
If you are using list supply as the basis for price, then you'd likely want to get rid of this condition.

```suggestion
     _listSupply[auth.key] += quantity;
```